### PR TITLE
feat: add notification page, add expo-notifications, configure expo p…

### LINKED ISSUE #13: Experiment: test notification for android
We want to test the Android notification for new app messages.
image

https://www.figma.com/file/HCoiw3jQvKt4vF3P1bZQMp/Wallet?type=whiteboard&node-id=134-955&t=wlzqNkuqJqofvewz-4

### PR DESCRIPTION
**Pull request by dagmawig**
…roject

fixes #13 

to send notification from expo dashboard to your android device
1) run the app on your physical android device (expo notifications doesn't work on emulator)
2) log in 
3) when you get to home page after log in, the device push token will be logged in the terminal
4) go to  expo dashboard using the link https://expo.dev/notifications
5) enter the device push token you got in step 3 into the "expo push token" field, fill out "title" and "body" section and hit send
6) you should receive notification on your android device with the title and body you entered in the expo dashboard.
7) when you click on the notification the app should direct you to the notification page of the app.

### CODE REVIEW

**Comment by dadiorchen** [comment]
@dagmawig is this ready for merge?

**Comment by dagmawig** [comment]
Yes it's ready. I have gone through the change I made with @pierrelstan

File: app.json [hunk 1]
```diff
@@ -25,6 +25,11 @@
     },
     "web": {
       "favicon": "./assets/favicon.png"
+    },
+    "extra": {
+      "eas": {
+        "projectId": "17ac480f-51f3-4acb-9b95-8b1bf6465495"
```
Comment (pierrelstan) [inline app.json:31]: In order to enhance the security and maintainability of the code, I suggest moving the `projectId` value from the code snippet into an environment variable. This practice will allow sensitive information to be stored separately from the codebase.

**Review by pierrelstan — APPROVED** [review]

**Comment by dadiorchen** [comment]
@pierrelstan can you merge this?

**Pull request merged**